tcptracer-bpf.c

```diff
@@ -780,8 +781,8 @@ __attribute__((always_inline))
 bool parse_http_response(char *buffer, int size, int *status_code_result) {
 	const char http_marker[4] = "HTTP";
 	if (size > 11) {
-		if (memcmp(buffer, http_marker, sizeof(http_marker)) == 0) {
-			int status_code = 100 * (buffer[9] - '0') + 10 * (buffer[10] - '0') + (buffer[11] - '0');
+		if (memcmp(buffer, http_marker, 4) == 0) {
```

Comment:
i'd keep sizeof()

Reply:
That was debug code :fearful: I'll change it back!